tests/test_upload.py

```diff
@@ -54,9 +54,9 @@ def test_upload(self):
             gdf = geopandas.GeoDataFrame(
                 df, geometry=geopandas.GeoSeries.from_wkt(polygons), crs="EPSG:4326")
 
-            dataset_id = ge.upload_dataframe(gdf)
+            dataset_name = ge.upload_dataframe(gdf)
 
-            self.assertEqual(dataset_id, DatasetId("fc5f9e0f-ac97-421f-a5be-d701915ceb6f"))
+            self.assertEqual(dataset_name, DatasetName("fc5f9e0f-ac97-421f-a5be-d701915ceb6f"))
```

Comment:
isn't this missing the namespace?

Reply:
[9d605a1](https://github.com/geo-engine/geoengine-python/pull/136/commits/9d605a196010ee301d1bd4ca0bdc30b823bdf44a)